content/en/docs/concepts/scheduling-eviction/topology-spread-constraints.md

```diff
@@ -635,6 +635,22 @@ section of the enhancement proposal about Pod topology spread constraints.
   You can work around this by using a Node autoscaler that is aware of
   Pod topology spread constraints and is also aware of the overall set of topology
   domains.
+- Node label typos can cause unexpected scheduling behavior. If a node has a typo
+  in its topology label (for example, `zon` instead of `zone`), the scheduler treats
+  that node as not having the topology key at all. This can result in:
+  - Pods failing to schedule with `UnschedulableAndUnresolvable` errors
+  - Uneven pod distribution that appears to violate spread constraints
+
+  Always verify that node labels exactly match the `topologyKey` specified in your
+  spread constraints.
```

Comment:
Is it really worth mentioning on the doc? Typo could lead to any unintended behavior in scheduling (or actually anything in Kubernetes).

Reply:
I prefer removing this section